upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -601,6 +603,10 @@ func (_ *LoadBalancer) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *LoadBalan
 		}
 	}
 
+	if err := t.AddELBTags(loadBalancerName, e.Tags); err != nil {
```

Comment:
I'm trying to think if running AddELBTags twice could be an issue, normally I'd make a local variable then add all tags to it.  Any reason you decided to run it twice?

Reply:
Perfect thanks!